client/galaxy/scripts/onload.js

```diff
@@ -183,4 +184,14 @@ $(document).ready( function() {
             }
         }
     }
+
+    // Load all webhooks with the type 'onload'
```

Comment:
This should be modified to only load webhooks when they are enabled.  The base application can pass this information through.  I disabled webhooks to test, and on a load of the base page three extra requests are made to the server to fetch webhooks currently.  (of course, they all return an empty [], but we shouldn't be making those requests if webhooks are not enabled)